### PR TITLE
DESCW-721 Header footer templates + parts

### PR DESCRIPTION
**Pull request by codewisenate**
- Updated template parts needed to create a global header and footer
- Updated all page templates to use global header/footer containers
- Updated theme.json

### CODE REVIEW

File: parts/global-header-container.html [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<!-- wp:group {"className":"bcgov-header-container","layout":{"type":"constrained","wideSize":"100%"}} -->
+<div class="wp-block-group bcgov-header-container"></div>
```
Comment (ShawnTurple) [inline parts/global-header-container.html]: are we not putting defaults in here?

File: parts/skip-nav.html [hunk 2]
```diff
@@ -1,13 +1,15 @@
...
+    <div class="wp-block-button skip-nav is-style-fill"><a class="wp-block-button__link has-headline-color-color has-background-background-color has-text-color has-background wp-element-button" href="#post-content">Skip to Main Content</a></div>
     <!-- /wp:button -->
     
-    <!-- wp:button {"className":"is-style-outline skip-nav"} -->
-    <div class="wp-block-button is-style-outline skip-nav"><a class="wp-block-button__link" href="#main-menu">Skip to Navigation</a></div>
+    <!-- wp:button {"backgroundColor":"background","textColor":"headline-color","className":"skip-nav is-style-fill","iconsList":[{"label":"None","value":""},{"label":"Default","value":"default"},{"label":"Home","value":"home"},{"label":"Auto","value":"auto"},{"label":"Book","value":"book"},{"label":"Calendar","value":"calendar"},{"label":"Case","value":"case"},{"label":"Changes","value":"changes"},{"label":"Federal","value":"federal"},{"label":"Gear","value":"gear"},{"label":"Group","value":"group"},{"label":"Leaf","value":"leaf"},{"label":"Map","value":"map"},{"label":"Tag","value":"tag"}]} -->
```
Comment (ShawnTurple) [inline parts/skip-nav.html:7]: just wondering what the different labels here are for, ie default, auto, calendar, changes, gear, leaf, tag.

**Review by ShawnTurple — APPROVED** [review]

File: parts/skip-nav.html (hunk 2 above)
Comment (codewisenate) [reply parts/skip-nav.html:7]: It's meta from the icon block style filter.

File: parts/global-header-container.html (hunk 1 above)
Comment (codewisenate) [reply parts/global-header-container.html]: Missing template part meta has been added. Good catch.

File: inc/core/patterns/bcgov-sequence-steps.php [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+namespace Bcgov\Theme\Block;
+
+/**
+ * Step through sequence pattern.
+ *
+ * @since 1.0.4
+ *
+ * @package Bcgov/Theme/Block
+ */
+return [
```
Comment (ShawnTurple) [inline inc/core/patterns/bcgov-sequence-steps.php:12]: this doesn't seem like a header or footer template.

**Review by ShawnTurple — APPROVED** [review]
try to avoid adding more code than the ticket indicates into the PR.

File: inc/core/patterns/bcgov-sequence-steps.php (hunk 3 above)
Comment (codewisenate) [reply inc/core/patterns/bcgov-sequence-steps.php:12]: Apologies, it should have been separated out.

**Pull request merged**